### PR TITLE
SGGame: Add proper information for USUM stuff

### LINKED ISSUE #68: SGgame - USUM Pokemon Catch Rates & Base EXP values
Catch rates/ Base EXP values currently aren't avaliable for the new USUM pokemon.
I've set temporary rates & values for them atm, but true data is wanted ASAP.

### PR DESCRIPTION
**Pull request by DeathlyPlays**
Closes #68

### CODE REVIEW

File: config/SGGame/pokemon.json [hunk 1]
```diff
@@ -17426,12 +17426,40 @@
...
-		"inherit": "necrozma"
+		"inherit": "necrozma",
+    "evDrops": {
+			"hp": 0,
+			"atk": 0,
+			"def": 0,
+			"spa": 3,
+			"spd": 0,
+			"spe": 0
+		}
```
Comment (HoeenCoder) [inline config/SGGame/pokemon.json]: Specify totalEvDrops even though it matches necrozma, incase necrozmas changes so it dosent match its other formes anymore.

File: config/SGGame/pokemon.json [hunk 2]
```diff
@@ -17426,12 +17426,40 @@
...
+    "id": "necrozmaultra",
+    "inherit": "necrozma",
+    "evDrops": {
+			"hp": 0,
+			"atk": 1,
+			"def": 0,
+			"spa": 1,
+			"spd": 0,
+			"spe": 1
+		}
```
Comment (HoeenCoder) [inline config/SGGame/pokemon.json]: this is battle only isnt it?

File: config/SGGame/pokemon.json (hunk 1 above)
Comment (DeathlyPlays) [reply config/SGGame/pokemon.json]: Ok

File: config/SGGame/pokemon.json (hunk 2 above)
Comment (DeathlyPlays) [reply config/SGGame/pokemon.json]: yes

File: config/SGGame/pokemon.json [hunk 3]
```diff
@@ -15841,33 +15841,33 @@
...
+		"baseExp": 189,
 		"evDrops": {
 			"hp": 0,
 			"atk": 0,
 			"def": 0,
 			"spa": 0,
-			"spd": 1,
-			"spe": 0
+			"spd": 0,
+			"spe": 1
```
Comment (HoeenCoder) [inline config/SGGame/pokemon.json]: Its 1 spd point, not speed

Comment (DeathlyPlays) [reply config/SGGame/pokemon.json]: http://prntscr.com/hcflup Actually it is Speed

**Pull request merged**